src/Package/Impl/Shell/VsAppShell.cs

```diff
@@ -220,6 +220,13 @@ public sealed class VsAppShell : IApplicationShell, IMainThread, IIdleTimeServic
             return new ProgressBarSession(vsProgressBarSession, vsProgressBarSession.UserCancellationToken);
         }
 
+        public ProgressBarSession ShowProgressBarWithUpdate(string waitMessage, int delayToShowDialigMs = 0) {
```

Comment:
Small typo here and in function above: delayToShowDialigMs -> delayToShowDialogMs